addons/pos_restaurant/models/pos_restaurant.py

```diff
@@ -100,6 +100,12 @@ class RestaurantTable(models.Model):
     color = fields.Char('Color', help="The table's color, expressed as a valid 'background' CSS property value", default="#35D374")
     parent_id = fields.Many2one('restaurant.table', string='Parent Table', help="The parent table if this table is part of a group of tables")
     active = fields.Boolean('Active', default=True, help='If false, the table is deactivated and will not be available in the point of sale')
+    display_name = fields.Char(string="Name", compute="_compute_display_name", readonly=False)
```

Comment:
Check this: https://github.com/odoo/odoo/commit/c2d47b976d1ac0eba6bd16add3d54f84e5e2d3d3#diff-c4cf79d0ff0cb8f7ca865fdfac774ce8fd95c800471a0932d126acb6fd8e7c27L139

Seems like declaring the field `display_name` isn't necessary. Can you try?

Reply:
:+1: I removed the declaration